README.md

```diff
@@ -8,17 +8,25 @@ PokeReader is a 3gx plugin that allows viewing information about 3ds Pokemon gam
 
 This information can be used to RNG shiny and high IV Pokemon, similar to lua scripts on emulators.
 
+## Commands
+- X+Y to Lock/Unlock
```

Comment:
The other lines have spaces around the `+`, so let's add that here for consistency: `X + Y`.

Also, we might consider specifying that this locks and unlocks inputs to remove ambiguity around what is being locked/unlocked.